### ISSUE TITLE
Sidebar on Account page missing gutters

### ISSUE THREAD

**Issue by lindsayevans** [payloadcms/payload#3772]
### Link to reproduction

https://github.com/boomworks/payload-plugin-author-fields/tree/feature/sidebar-layout-issue/demo

### Describe the Bug

Fields on the Users collection with `position: 'sidebar'` get displayed below the main form on Account page, without any left/right gutters.
Displays correctly on the Users page.

Found while investigating [an issue on payload-plugin-author-fields](https://github.com/boomworks/payload-plugin-author-fields/issues/11) - haven't done a full reproduction, but the link is to a branch showing the issue.

### To Reproduce

1. Add a new field to Users collection with `position: 'sidebar'`
```
{
  name: 'jobTitle',
  type: 'text',
  admin: {
    position: 'sidebar',
  },
},
```
2. Browse to your account page (/admin/account)
3. Field is displayed below the main form, without space on the left side:
<img width="435" alt="image" src="https://github.com/payloadcms/payload/assets/26568/a46e2a9d-be5f-4536-812b-6ac9091e2bcd">

### Payload Version

2.0.11

**Comment by jacobsfletch** [comment]
Thank you the report on this @lindsayevans 🙏  I've got a fix queued up in #3798

**Comment by lindsayevans** [comment]
Awesome, thanks!
